node/get_status_node.go

```diff
@@ -219,32 +220,35 @@ func (n *StatusNode) StartLocalBackup() error {
 		}
 	}
 
+	chatAccount, err := n.gethAccountsManager.SelectedChatAccount()
+	if err != nil {
+		return err
+	}
+
+	privateKey := chatAccount.PrivateKey()
+
 	filenameGetter := func() (string, error) {
-		profileKeypair, err := n.accountsSrvc.GetProfileKeypair()
+		backupPath, err := n.accountsSrvc.GetBackupPath()
 		if err != nil {
 			return "", err
 		}
 
-		backupPath, err := n.accountsSrvc.GetBackupPath()
+		compressedPubKey, err := utils.SerializePublicKey(crypto.CompressPubkey(&privateKey.PublicKey))
 		if err != nil {
 			return "", err
 		}
+
 		var backupDir string
 		if backupPath != "" {
 			backupDir = backupPath
 		} else {
 			backupDir = filepath.Join(n.config.RootDataDir, "backups")
 		}
-		fullPath := filepath.Join(backupDir, fmt.Sprintf("%x_user_data.bkp", profileKeypair.KeyUID[len(profileKeypair.KeyUID)-4:]))
-		return fullPath, nil
-	}
 
-	chatAccount, err := n.gethAccountsManager.SelectedChatAccount()
-	if err != nil {
-		return err
-	}
+		fullPath := filepath.Join(backupDir, fmt.Sprintf("%s_user_data.bkp", compressedPubKey[len(compressedPubKey)-6:]))
 
-	privateKey := chatAccount.PrivateKey()
+		return fullPath, nil
+	}
 
 	n.localBackup, err = backup.NewController(backup.BackupConfig{
 		PrivateKey:     crypto.Keccak256(crypto.FromECDSA(privateKey)),
```

Comment:
Changes in this PR are ok, but I want to comment this `PrivateKey` field, which is misleading and confused me here cuase `crypto.Keccak256(crypto.FromECDSA(privateKey))` is not a private key for sure, could we rename it to `EncryptionKey`?

That means `backup.Controller` and `backup.core` types work with `EncryptionKey`, which is provided via `backup.BackupConfig`. And it's on user to determine if encryption key will be ` crypto.Keccak256(crypto.FromECDSA(privateKey))` or anything else. We could also use the same encryption we use in case of keycard profile, that's a PublicKey of `m/43'/60'/1581'/1'/0` derivation of profile key pair.